service/resource/configmap/desired_test.go

```diff
@@ -696,366 +696,6 @@ func Test_Resource_ConfigMap_GetDesiredState(t *testing.T) {
 			},
 			expectedErrorHandler: nil,
 		},
-
```

Comment:
The following two tests are scrape config styles that we no longer support, and aren't present in installations anymore.